.gitignore

```diff
@@ -33,6 +34,7 @@ build-cs/composer.lock
 node_modules/
 vendor/
 
+modules/js-and-css/partytown-web-worker/assets/js/partytown
```

Comment:
This will necessitate introducing a `.distignore` file to replace the `.gitattributes` file as explained in https://github.com/WordPress/performance/issues/893#issuecomment-1947398713.